common/meta/sled-store/src/sled_tree.rs

```diff
@@ -547,6 +557,22 @@ impl<'a, KV: SledKeySpace> AsTxnKeySpace<'a, KV> {
     ) -> Result<Option<KV::V>, UnabortableTransactionError> {
         self.inner.insert::<KV>(key, value)
     }
+
+    pub fn get(&self, key: &KV::K) -> Result<Option<KV::V>, UnabortableTransactionError> {
+        self.inner.get::<KV>(key)
+    }
+
+    pub fn remove(&self, key: &KV::K) -> Result<Option<KV::V>, UnabortableTransactionError> {
+        self.inner.remove::<KV>(key)
+    }
+}
+
+impl<'a, KV: SledKeySpace> Deref for AsTxnKeySpace<'a, KV> {
+    type Target = &'a TransactionSledTree<'a>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.inner
+    }
```

Comment:
It would be better if there are some comments explaining why need to deref it.
Since the deref-ed instance behave differently from the original one.